virgo/ui/pages/canvas_page.py

```diff
@@ -42,89 +42,119 @@ def Page(self: App):
+    scrollbar_config.grid(row=0, column=1, sticky="ns")
+    
+    # Configure the canvas to work with the scrollbar
+    canvas_config.configure(yscrollcommand=scrollbar_config.set)
+    
+    # Create a frame within the canvas to hold the actual configurations
+    configFrame = tk.Frame(canvas_config)
+    canvas_config.create_window((0, 0), window=configFrame, anchor="nw")
+
+    # Populate configFrame with the scrollable content
     def build_configurations():
         utils.destroy_children(configFrame)
-        ttk.Label(configFrame, text="Prebuilt Configurations", style="H3.TLabel").grid()
+        ttk.Label(configFrame, text="Prebuilt Configurations", style="H3.TLabel").pack(anchor="w", pady=5)
+        
         try:
-            #TODO:  Dont hard code this for max & linux
             configPaths = os.listdir("configurations")
         except Exception as e:
             print(e)
             configPaths = []
+
         for path in configPaths:
             if ".virgo" in path:
-                with open(os.path.join("configurations",path), "r") as file:
-                    #TODO: redundant load> low priority
+                with open(os.path.join("configurations", path), "r") as file:
                     json_data = file.read()
                     data = json.loads(json_data)
                     title = data["metadata"]["title"] if "metadata" in data and "title" in data["metadata"] else "Untitled"
                     description = data["metadata"]["description"] if "metadata" in data and "description" in data["metadata"] else "No description"
-                ttk.Button(configFrame,text="{}".format(title), command=lambda x=f"configurations/{path}": self.load_canvas(x)).grid()
-                ttk.Label(configFrame, text=f"info: {description}\n@ {path}").grid()
+
+                ttk.Button(configFrame, text="{}".format(title), command=lambda x=f"configurations/{path}": self.load_canvas(x)).pack(anchor="w", pady=5)
+                ttk.Label(configFrame, text=f"info: {description}\n@ {path}").pack(anchor="w", pady=5)
+
     build_configurations()
-    def scroll_handler(event):
-        tabIdx = panel.index(panel.select())
-        if tabIdx == 0:
-            widgets.on_mousewheel(event)
-        elif tabIdx == 1:
-            config.on_mousewheel(event)
 
-    widgets.build(master_scroll=scroll_handler)
-    config.build(master_scroll=scroll_handler)
+    # Update the scroll region after adding configurations
+    configFrame.update_idletasks()
+    canvas_config.config(scrollregion=canvas_config.bbox("all"))
     
-    widgets.grid(column=0, row=0, sticky="nsew")
-    config.grid(column=0, row=0, sticky="nsew")
+    # Bind the scroll functionality to the mouse wheel
+    canvas_config.bind_all("<MouseWheel>", on_mousewheel)
+
+    # Configure the configMenu frame to expand
+    self.configMenu.pack(fill="both", expand=True, padx=10, pady=10)
 
     def tab_change_handler(event):
```

Comment:
there seems to be an issue on mac when changing tabs that the content in the configurations tabs will not load until the mouse is moved over it. Does this behavior reproduce for you?

Reply:
This is weird. I haven't encountered that issue on my Mac. The content in the configuration tabs loads immediately when switching between tabs.